### DISCUSSION TITLE
How do I change the path to config.yaml

### DISCUSSION THREAD

**Discussion by B1tWhys** [mitmproxy/mitmproxy#6785] [Q&A] [answered]
Hi,

I'd like to use a `config.yaml` file that's not in [the default location](https://docs.mitmproxy.org/stable/concepts-options/). I feel like there's gotta be an option right in front of my face for this, but I can't seem to find it in the `--help`, or in the docs. Can I get a hint?

If it matters (I can't imagine it does) I'm on:
```
Mitmproxy: 10.0.0
Python:    3.11.8
OpenSSL:   OpenSSL 3.2.1 30 Jan 2024
Platform:  macOS-13.1-arm64-arm-64bit
```

**Comment by Prinzhorn** [comment, marked as answer]
[`confdir`](https://docs.mitmproxy.org/stable/concepts-options/#confdir) (refs https://github.com/mitmproxy/mitmproxy/issues/6528)

**Reply by B1tWhys** [reply]
Ah, thanks. I figured it must be there somewhere...

**Reply by God-damnit-all** [reply]
@Prinzhorn Is there an environment variable equivalent? (If not, there should be!)

**Reply by Prinzhorn** [reply]
Looks like you found https://github.com/mitmproxy/mitmproxy/issues/7591, leaving it here for others to find
